dpl-config.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
is this file needed ?

Reply:
I have no idea, I do not even know how it got there. I have removed it.